### PR TITLE
[MaxScale] Handle config boolean parameters

### PR DESCRIPTION
**Pull request by nervo**

### CODE REVIEW

File: manala.maxscale/templates/config/_macros.j2 [hunk 1]
```diff
@@ -52,6 +52,10 @@
 
 {%- macro _config_parameter_value(value) -%}
     {%- if value is sameas none -%}
+    {%- elif value is sameas true -%}
+        true
+    {%- elif value is sameas false -%}
+        true
```
Comment (chalasr) [inline manala.maxscale/templates/config/_macros.j2]: false?

Comment (nervo) [reply manala.maxscale/templates/config/_macros.j2]: oh

Comment (nervo) [reply manala.maxscale/templates/config/_macros.j2]: my

Comment (nervo) [reply manala.maxscale/templates/config/_macros.j2]: god

**Pull request merged**